Sources/LocalReceiptParsing/Builders/ASN1ContainerBuilder.swift

```diff
@@ -55,6 +51,9 @@ private extension ASN1ContainerBuilder {
         while currentPayload.count > 0 {
             let internalContainer = try build(fromPayload: currentPayload)
             internalContainers.append(internalContainer)
+            if internalContainer.containerIdentifier == .endOfContent {
+                break
+            }
```

Comment:
this makes sense as a general thing, for all containers. But only indefinite-length containers seem to have this
¯\\_(ツ)_/¯

In any case, I don't think we need to check whether the length is indefinite, since it still makes sense to stop if you reach end of content